Dockerfile

```
@@ -0,0 +1,27 @@
+FROM debian:12-slim
+
+RUN apt update && \
+    apt upgrade && \
+    apt install -y --no-install-recommends \
+      aha \
+      bash \
+      bind9-dnsutils \
+      curl \
+      grepcidr \
+      ipcalc \
+      jq \
+      mtr-tiny \
+      ncat \
+      ncurses-bin \
+      nmap \
+      whois \
```

Running `asn` using this image for an IP gave me the following output:
```
$ docker run -it --rm asn 104.16.x.x

────────────────────────────────────────────────────────────
                        WARNING

No IPQualityScore token found, so disabling in-depth threat
analysis and IP reputation lookups. Please visit
https://github.com/nitefood/asn#ip-reputation-api-token
for instructions on how to enable it.
────────────────────────────────────────────────────────────

╭──────────────────────────────╮
│ ASN lookup for 104.16.x.x│
╰──────────────────────────────╯
Collecting pWhois data (press CTRL-C to cancel)...getaddrinfo(whois.pwhois.org): Servname not supported for ai_socktype
[...]
```

It turns out `whois` requires the `netbase` package, 

After adding it, the ASN lookup completed successfully:
```suggestion
      whois \
      netbase \
```